implementations/rust/ockam/ockam_app/src/invitations/tray_menu.rs

```diff
@@ -115,24 +102,24 @@ fn add_received_menu<R: Runtime>(
     app_handle: &AppHandle<R>,
     received: &[ReceivedInvitation],
 ) -> Submenu<R> {
-    debug!(count = received.len(), "Received invitations");
-
-    let header_text = if received.is_empty() {
-        "No Received Invitations"
-    } else {
-        "Received Invitations"
-    };
-
-    let mut submenu_builder =
-        SubmenuBuilder::with_id(app_handle, INVITATIONS_RECEIVED_HEADER_MENU_ID, header_text);
-
+    debug!(
+        count = received.len(),
+        "Building menu for received invitations"
+    );
+
+    let header_text = format!(
```

Comment:
I suspect that this kind of code should be put in a `text` directory / file somewhere for text manipulation with an appropriate `pluralize` function. I've had to do that quite a few times to display quantities in various libraries.

Reply:
Noted! I'll tackle this in a separate PR